### PR TITLE
[BLOG] Spyder 2022 roadmap and new grants blog post

### PR DESCRIPTION
**Pull request by dalthviz**
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

Hi! I think the post is following the guidelines but any comment, suggestion or correction is greatly appreciated. Also, I'm not totally sure if adding images (the `featureImage` and `hero` images) is required but if that is the case let me know :)

### CODE REVIEW

**Review by gabalafou — CHANGES_REQUESTED** [review]
Hey @dalthviz , sorry it took us so long to get to this! We honestly didn't even see it until today because of all of the madness around getting the site off the ground for the SciPy conference.

I left some review comments. Many of them are minor corrections or suggestions for English grammar. Feel free to accept or ignore the suggestions wherever you think best. :)

The one possible blocker that we need to make a decision on is the lack of a hero image in the yaml top section. Without a hero image, the blog post does not quite render correctly.



It's not easy to see in the screenshot above but there is a little bump poking out from the bottom of the nav bar. That's the "Back to blog" link. It's hidden underneath the nav bar because the markup expects there to be a hero image.

I think we have two choices:

1. Use one of the stock hero images. @noatamir can you chime in where Daniel can find those.
2. Find something that can be used as a hero image specifically for this blog post.

It would also be nice to have a featured image because without it, the link to this post looks a little sad next to the other links that have images:

File: apps/labs/posts/spyder-roadmap-proposals-2022.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+With this grant we were able to contract one new developer, Juan Sebastian Bautista, a Systems and Computing engineer recent graduate from Universidad de los Andes in Bogotá, Colombia. Juan Sebastian will help us implement the proposal working alongside with the Spyder team in a six months time frame. For more details about this grant you can check the [Spyder wiki page for the proposal](https://github.com/spyder-ide/spyder/wiki/NumFOCUS-Small-Development-Grant-2022).
+
+## Next steps
+
+We will be focusing our time as a project on pushing forward the accepted proposals and submitting new ones. On top of this we will try to get more feedback about the things that volunteers, users and community in general want to see in the future for Spyder and share the proposals progress relaunching the [Spyder Community calls](https://github.com/spyder-ide/community#spyder-communitycalls).
```
[the remark this answers is not in the document]
Comment (dalthviz) [reply apps/labs/posts/spyder-roadmap-proposals-2022.md:28]: Yep exactly! I will try to rephrase this to be more clear 👍🏼. Also, thank you so much for taking the time to review this! :)

**Comment by dalthviz** [comment]
Thank you again @gabalafou for the review! Thinking about the featured and hero images could something like the Spyder logo (for the featured image) and banner (for the hero image) work?

spyder-title


spyder_readme_banner

**Comment by gabalafou** [comment]
Those images seem fine to me, but I would like a second opinion from @noatamir.

**Comment by gabalafou** [comment]
@dalthviz, there's [additional guidance (see number 7) on these blog images](https://www.notion.so/quansightlabs/Blogging-at-Quansight-Labs-4459748222394f2084dc7f609904a383) in Notion.

**Comment by noatamir** [comment]
> Those images seem fine to me, but I would like a second opinion from @noatamir.

LGTM

**Comment by noatamir** [comment]
@gabalafou the images are the correct size, but they don't appear in the preview link. 
I had a similar question on #406. 
Is this ops related?

**Comment by trallard** [comment]
Images are looking good and the content seems to be all good. @dalthviz @noatamir  am I ok to merge this?

**Review by trallard — APPROVED** [review]

File: apps/labs/posts/spyder-roadmap-proposals-2022.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+title: 'Spyder 2022 roadmap and new grants work'
+published: July 12, 2022
```
Comment (trallard) [inline apps/labs/posts/spyder-roadmap-proposals-2022.md:3]:
```suggestion
published: November 8, 2022
```

@dalthviz  I am changing the date so that this goes into the next deployment
